core/src/toga/style/pack.py

```diff
@@ -118,6 +121,11 @@ def apply(self, prop, value):
                         weight=self.font_weight,
                     )
                 )
+            elif prop in (
+                "height",
+                "width",
+            ):
+                self._applicator.set_size(self.width, self.height)
```

Comment:
You're correct that this was added to deal with the bezel style. 

The approach of adding `set_size()` was the result of me overthinking this a bit. The concern I was trying to avoid is that macOS buttons (and buttons generally) *don't* expand vertically to fill space - and as a result, on cocoa, we only need the bezel change when the height is set explicitly. Adding `set_size()` meant we could respond directly to setting the height of the widget, rather than implicitly responding whenever the bounds are set. 

However, in practice, it won't actually change the observed behavior. Absent of a specific height value, the bounds height of a button will be the intrinsic height - the bounds of the button don't expand to fill the available space, even if the layout would allow it..